src/renderer/views/app/components/Toolbar/index.tsx

```diff
@@ -77,7 +77,7 @@ export const Toolbar = observer(() => {
         {hasCredentials && (
           <ToolbarButton icon={icons.key} size={16} onClick={onKeyClick} />
         )}
-        {!isWindow && (
+        {!isWindow && store.options.object.shield == true && (
```

Comment:
Isn't it `store.settings.object.shield`?

Reply:
done